tests/pkg_micro-ecc/tests/01-run.py

```diff
@@ -4,16 +4,19 @@
 from testrunner import run
 
 
-# Use a custom global timeout for slow hardware. On microbit (nrf51), the
-# test completes in 80s.
+# Use a custom global timeout for slow hardware. On ATmegas clocked at 8MHz
+# one test round completes in ~36s
 TIMEOUT = 100
 
 
 def testfunc(child):
     child.expect_exact('micro-ecc compiled!')
-    child.expect_exact('Testing 16 random private key pairs and signature '
+    child.expect_exact(r'Testing (\d+) random private key pairs and signature '
```

Comment:
```suggestion
    child.expect(r'Testing (\d+) random private key pairs and signature '
                 'without using HWRNG')
```
Sorry, the automatic test doesn't work without this change. You can squash directly.

Reply:
Done. Thanks for testing!